Doc/library/datetime.rst

```diff
@@ -1316,56 +1316,61 @@ Examples of working with datetime objects:
 
 Using datetime with tzinfo:
 
-    >>> from datetime import timedelta, datetime, tzinfo
-    >>> class GMT1(tzinfo):
+    >>> from datetime import timedelta, datetime, tzinfo, timezone
+    >>> class KabulTz(tzinfo):
```

Comment:
If you add:

```python
def __repr__(self):
    return f"{self.__class__.__name__}()"
```
The later examples where you attach this to a datetime will be easier to read, e.g.:

```python
   >>> dt2     # doctest: +ELLIPSIS
   datetime.datetime(2006, 6, 14, 13, 0, tzinfo=KabulTz())
```

You may also consider adding an `__str__` that returns the tzdata IANA key (though I believe this is used nowhere in your examples later):

```python
def __str__(self):
    return "Asia/Kabul"
```